### PR TITLE
lesson 3 - done.

### PR DESCRIPTION
**Pull request by AlexeyTretyakov98**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+//     str2 = str2.reverse();
+//     str2 = str2.join('');
+//     if (str == str2) {
+//         return true;
+//     }
+//         else return false;
+//     {
+//     }
+// }
+// console.log(palindrome(''))
```
Comment (Marinarium) [inline js/main.js]: Why did you keep this code?

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function palindrome(str) {
+  return str == str.split('').reverse().join('');
+}
+console.log(palindrome('шалаш'));
+
+let nums = [12, 53, 20, 18, 22, 100, 43, 57, 50, 1];
+let str1 = nums.toString();
+str1 = str1.split('0').join('zero');
+let numsNew = str1.split(',');
```
Comment (Marinarium) [inline js/main.js]: Your way returns an array of strings` ["12", "53", "2zero", "18", "22", "1zerozero", "43", "57", "5zero", "1"]`
According to the task, you must return array of numbers and strings `[12, 53, '2zero', 18, 22, '1zerozero', 43, 57, '5zero', 1]`

And it needs to create a function, that we can use for every array, I see that your decision works only for one array. 

And I've not found this part: создайте массив с десятью случайными элементами от 0 до 100;

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (AlexeyTretyakov98) [reply js/main.js]: I have kept for myself the full version of the abbreviated form

File: js/main.js (hunk 2 above)
Comment (AlexeyTretyakov98) [reply js/main.js]: fix

File: js/main.js (hunk 1 above)
Comment (Marinarium) [reply js/main.js]: Just remember leaving something "for myself" is unacceptable on real projects

**Pull request merged**